internal/handler/msg_getmore.go

```diff
@@ -156,6 +160,14 @@ func (h *Handler) MsgGetMore(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		)
 	}
 
+	if maxTimeMSPresent && c.Type != cursor.TailableAwait {
```

Comment:
Hm, strange that codecov shows the lines below as not covered

Reply:
I had similar issue with previous PR, I think that codecov reports might be returned too early?